pkg/actors/actors.go

```diff
@@ -800,25 +803,32 @@ func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderR
 		RegisteredTime: time.Now().UTC().Format(time.RFC3339),
 	}
 
-	reminders, err := a.getRemindersForActorType(req.ActorType)
-	if err != nil {
-		return err
-	}
+	err := backoff.Retry(func() error {
+		reminders, remindersEtag, err := a.getRemindersForActorType(req.ActorType)
+		if err != nil {
+			return err
+		}
 
-	reminders = append(reminders, reminder)
+		reminders = append(reminders, reminder)
```

Comment:
Unrelated to your changes but is this right? I'm comparing this code with: https://github.com/dapr/dapr/pull/2848/files#diff-aeb183ea3b81464cb579aa3e309be6bbfa88d6d900cfdbddb8c74d89ee7d4dc6R994

I find it interesting (and subtle) that this code unconditionally adds a reminder per-call, whereas delete unconditionally deletes **all** reminders with the specified ID and name.

Reply:
There cannot be two reminders with the same name and actorId, so the behavior end up being the same.

Reply:
> There cannot be two reminders with the same name and actorId, so the behavior end up being the same.

where is that enforced?

Reply:
Check line 765 in this file. It starts the check process in CreateReminder() function.